README.md

```diff
@@ -14,10 +18,57 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### Node.js
+
+If you want to install Node.js natively visit [Node.js](http://nodejs.org), please make sure it is version 4.4.4
+
+To check the locally installed version run: 
+
+```bash
+node --version
 ```
-git clone https://github.com/department-of-veterans-affairs/healthcare-application.git
-cd healthcare-application
+
+Once you have version 4.4.4 of Node.js installed to required version of npm run:
+
+```bash
+npm i -g npm@3.8.9
+```
+
+You may want to use nvm (Node Version Manager), this will allow for managing and installing multiple versions of Node.js on your machine.
```

Comment:
Either use 2 separate sentences, or say `You may want to use nvm, which will allow...`